etc/env/file_based/general.yaml

```
@@ -272,6 +272,23 @@ prover_group:
       aggregation_round: 1
     - circuit_id: 18
       aggregation_round: 1
+prover_job_monitor:
+  prometheus_port: 3317
+  max_db_connections: 9
+  graceful_shutdown_timeout_ms: 5000
+  gpu_prover_archiver_run_interval_ms: 86400000
+  gpu_prover_archiver_archive_prover_after_secs: 172800
+  prover_jobs_archiver_run_interval_ms: 1800000
+  prover_jobs_archiver_archive_jobs_after_secs: 172800
```

```suggestion
  prover_jobs_archiver:
    run_interval_ms: 1800000
    jobs_after_secs: 172800
```

another nit: Another layering of config could potentially improve readability here